pkgs/development/libraries/libraw1394/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation rec {
   meta = { 
     description = "Library providing direct access to the IEEE 1394 bus through the Linux 1394 subsystem's raw1394 user space interface";
     homepage = "https://ieee1394.wiki.kernel.org/index.php/Libraries#libraw1394";
-    license = ["GPL" "LGPL"];
+    license = stdenv.lib.licenses.lgpl2;
```

Comment:
I think the original [GPL LGPL] was wrong. The source only contains a copy of the LGPL, and I can't find any mention of it also being licensed under GPL anywhere. I believe the other ones I fixed were wrong as well.